### PR TITLE
Add capture future interface blog post

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

File: blog/capture-future-intf.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Once deployed we get :
+
+<p>
+  <a href="/assets/images/blog/capture-future-1.png" data-lightbox="WebUI-1" data-title="Skydive capture">
+    <img src="/assets/images/blog/capture-future-1.png"/>
+  </a>
+</p>
+
+Now it is time to create our capture. I'll use the Web UI but the command could be used as well.
```
Comment (safchain) [inline blog/capture-future-intf.md]: command line

File: blog/index.md [hunk 2]
```diff
@@ -3,6 +3,11 @@ title: Blog
 layout: blog-index
 ---
 
+# [Capture for interfaces that don't exist (yet)](/blog/capture-future-intf.html)
+## by Sylvain Afchain, 09/10/2018
+
+Since `Skydive` already has a `Python` client [library](/documentation/api-python) I thought it was "fun" to create an `Ansible` module leveraging it to add topology entities. In this blog post I will show how to use this module and how to use this module to provide real topology information.
```
Comment (masco) [inline blog/index.md]: description needs to change!

File: blog/capture-future-intf.md (hunk 1 above)
Comment (masco) [reply blog/capture-future-intf.md]: how about CLI instead of command line?

**Comment by lebauce** [comment]
```
index b2f96df..7f1e1ab 100644
--- a/blog/capture-future-intf.md
+++ b/blog/capture-future-intf.md
@@ -5,9 +5,9 @@ author: Sylvain Afchain
 date: 09/10/2018
 ---
 
-I have been reached out a couple of time for a feature request which was something like :
+I have been reached out a couple of times about a feature request which was something like :
 
-"I would like to capture the very first packets, like DHCP, of a VM that is about to boot. It will be great to add a mechanism to start a capture for a future interface."
+"I would like to capture the very first packets, like DHCP, of a VM that is about to boot. It would be great to add a mechanism to start a capture for a future interface."
 
 So yes it will be nice, and in fact it is already possible since the beginning of Skydive as it was one of the use cases that we wanted to address. Let's see how to achieve this.
 
@@ -16,7 +16,7 @@ So yes it will be nice, and in fact it is already possible since the beginning o
 As an example I'll use an `OpenStack` environment. You can have a look at the [Getting started](/documentation/getting-started#openstackdevstack) section to know how to have 
 `Skydive` deployed as part of `Devstack`.
 
-Once deployed we get :
+Once deployed we get:
 
 <p>
   <a href="/assets/images/blog/capture-future-1.png" data-lightbox="WebUI-1" data-title="Skydive capture">
@@ -37,9 +37,9 @@ The gremlin expression has to be read like :
 
 "Capture all the interface having the attribute ExtID.vm-id"
 
-In an OpenStack environment, VMs interfaces are plugged in an `OpenvSwitch` bridge and the `OpenvSwitch/Neutron` probes of Skydive reports the "VM ID" as `ExtID.vm-id` metadata attribute.
+In an OpenStack environment, VMs interfaces are plugged into an `OpenvSwitch` bridge and the `OpenvSwitch/Neutron` probes of Skydive reports the "VM ID" as `ExtID.vm-id` metadata attribute.
 
-Now that the capture is created, on the `Captures` list we can see our capture with a "Warning" icon. This icon indicates that the capture definition currently doesn't match any interface and consequently no packet capture has been started (yet).
+Now that the capture is created, we can see the `Captures` list our capture with a "Warning" icon. This icon indicates that the capture definition currently doesn't match any interface and consequently no packet capture has been started (yet).
 
 <p>
   <a href="/assets/images/blog/capture-future-2.png" data-lightbox="WebUI-2" data-title="Skydive capture">
```

**Pull request merged**